alignment.Dockerfile

```diff
@@ -0,0 +1,62 @@
+# syntax=docker/dockerfile:1
+FROM ubuntu:focal
+ENV DEBIAN_FRONTEND=noninteractive
+RUN <<EOF
+# build SDL2
```

Comment:
It is currently not clear whether "full headless mode" can be achieved with SDL2 (see https://github.com/EleutherAI/minetest/issues/46).
That's why I think there should be an option to skip building SDL2.